src/main/java/edu/kis/powp/jobs2d/command/manager/DriverCommandManager.java

```diff
@@ -40,11 +45,21 @@ public synchronized void setCurrentCommand(List<DriverCommand> commandList, Stri
 	 * @return Current command.
 	 */
 	public synchronized DriverCommand getCurrentCommand() {
-		return currentCommand;
+		return this.currentCommand;
+	}
+
+	public synchronized void setDriverManager(DriverManager driverManager) {
+		this.driverManager = driverManager;
+	}
+
+	@Override
+	public void runCommand() {
+		getCurrentCommand().execute(DriverFeature.getDriverManager().getCurrentDriver());
```

Comment:
This is an exaggeration. Using private fields internally usually does not require encapsulation.
Moreover, you introduce an unnecessary dependency (DriverFeature).